### DISCUSSION TITLE
Support for K33/34

### DISCUSSION THREAD

**Discussion by redarflap** [madMAx43v3r/chia-plotter#236] [Q&A]
Hi there,
are there plans for support of K33 and bigger plot sizes in the future?

**Comment by SebMoore** [comment]
Please check if your question has already been addressed before making a duplicate issue: see #25, #65, #217

**Reply by redarflap** [reply]
Alright, sorry.
I searched explicitely for K33 before but nothing showed up. I am not experienced with Github

**Reply by SebMoore** [reply]
Yeah, no worries. If you want to check, just type k33 and switch the results tab to "issues".
